libgnucash/engine/gnc-rational-rounding.hpp

```diff
@@ -98,7 +98,7 @@ round(T num, T den, T rem, RT2T<RoundType::promote>)
 {
     if (rem == 0)
         return num;
-    return num + (num < 0 ? -1 : 1);
+    return num + (rem < 0 ? -1 : 1);
```

Comment:
Need a comment explaining this. It will seem weird without context.

Reply:
What I did there was wrong and ended up checking the signs of the numerator and remainder to ascertain whether zero should be rounded up or down.